index.rst

```diff
@@ -41,7 +41,8 @@ visit our `gallery </docs/dev/auto_examples>`__.
 
         from skimage import data, io, filters
 
-        image = data.coins() # or any NumPy array!
+        # ... or any other NumPy array!
+        image = data.coins()
```

Comment:
I have tried that also, but it doesn't seem logical (comment usually address the line beneath).
Let's wait for more opinions.